s3torchconnectorclient/python/tst/integration/test_logging.py

```diff
@@ -0,0 +1,105 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import pytest
+
+PYTHON_TEST_CODE = """
+import logging
+import os
+import sys
+
+os.environ["PYTHONUNBUFFERED"] = "1"
+os.environ["ENABLE_CRT_LOGS"] = "{0}"
+
+from s3torchconnector import S3MapDataset
+
+logging.basicConfig(
+    stream=sys.stdout,
+    format="%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s",
+)
+logging.getLogger().setLevel(logging.INFO)
+
+if __name__ == "__main__":
```

Comment:
I'm not sure why this block is here - normally it's conjoined with a `main` function along with argument passing, or not present

Reply:
Meant to be "running as a script", but we can't import it as a module, as it is not a standalone file, dropping.